index.html

```diff
@@ -224,6 +243,10 @@
           <li>
             to invoke an <a>Action</a>: take the parameters from the request, execute the defined action, and return the result;
           </li>
+          <li>to observe a <a>Property</a>;</li>
```

Comment:
the listed items look fine. maybe we should group them. First properties, then action and lastly events...